src/Mobs/MagmaCube.h

```diff
@@ -19,10 +19,14 @@ class cMagmaCube :
 
 	virtual void GetDrops(cItems & a_Drops, cEntity * a_Killer = nullptr) override;
 	int GetSize(void) const { return m_Size; }
+
+	/** Returns the text describing the slime's size, as used by the client's resource subsystem for sounds.
+	Returns either "big" or "small". */
+	const AString GetSizeName(int a_Size) const;
 	
 protected:
 
-	/// Size of the MagmaCube, 1 .. 3, with 1 being the smallest
+	/// Size of the MagmaCube, 1, 2 and 4, with 1 being the smallest
```

Comment:
Even in vanilla the size can be really any number, it just happens that vanilla spawns only sizes 1, 2 and 4.